libbuteosyncfw/profile/SyncResults.h

```diff
@@ -82,6 +82,7 @@ class SyncResults
         //These error codes are mapped to syncAgentConsts.h
         // Successful 3xx
         SYNC_FINISHED = 301,
+        SYNC_FINISHED_WITH_ITEM_ERRORS,
```

Comment:
the syncAgentConsts.h is from buteo-syncml plugin, I believe.
That plugin seems to treat SYNC_FINISHED as "pure success" so I would personally lean toward adding this new SYNC_FINISHED_WITH_ITEM_ERRORS just to be explicit in this case.  But I might be wrong.